py/picca/data.py

```diff
@@ -512,6 +505,14 @@ def __init__(self,
         if reso is not None:
             reso = rebin_reso[w] / rebin_ivar[w]
 
+        # apply dust extinction correction
```

Comment:
I guess this is just a reordering so corrections are more close together?

Reply:
Mostly yes, but there are some minor changes to the deltas if this correction is applied before rebinning.

Reply:
yes, that's what you always talked about when you said that corrections are now a different ordering, but this routine will become deprecated anyway... Maybe we should generally add DeprecationWarnings to picca_deltas and routines that only this uses....